ports/rp2/mphalport.c

```diff
@@ -268,3 +268,11 @@ void soft_timer_init(void) {
 void mp_wfe_or_timeout(uint32_t timeout_ms) {
     best_effort_wfe_or_timeout(delayed_by_ms(get_absolute_time(), timeout_ms));
 }
+
+int mp_hal_is_pin_reserved(int n) {
+    #if MICROPY_PY_NETWORK_CYW43
+    return n == CYW43_PIN_WL_HOST_WAKE;
```

Comment:
CYW43_PIN_WL_REG_ON?

Reply:
Not needed as all this is used for is to avoid disabling irqs in machine_pin_deinit on a soft reset. Host wake is the only gpio that has this.